### PR TITLE
* use upload progress notification

### PR DESCRIPTION
**Pull request by mritunjayz**
* use upload progress notification
* changes in position for mobile view
* voice wave animation in leaderboard

### CODE REVIEW

**Comment by mritunjayz** [comment]
WIP

**Comment by mritunjayz** [comment]
Ready

File: web/src/components/pages/dashboard/stats/leaderboard.css [hunk 1]
```diff
@@ -335,6 +335,18 @@
...
+        .lottie {
+            position: relative;
+            height: 32px;
+            width: 600px;
+            left: -10px;
+            bottom: 5px;
+            z-index: 1000;
+        }
+
+        .aniamtion {
```
Comment (Gregoor) [inline web/src/components/pages/dashboard/stats/leaderboard.css]: dead code is dead

File: web/src/components/pages/profile/avatar-setup/avatar-setup.css [hunk 2]
```diff
@@ -26,18 +26,19 @@
...
+        right: 15%;
         padding-top: 10px;
     }
     .group-1 {
         width: 508px;
         height: 169px;
         object-fit: contain;
         background-image: url('./group-1.svg');
+        background-repeat: no-repeat;
         @media only screen and (max-width: 375px) {
```
Comment (Gregoor) [inline web/src/components/pages/profile/avatar-setup/avatar-setup.css]: I don't think we need the media query at all here, see this example: https://jsfiddle.net/zk2wnLpj/
We just define a max-width and when you shrink the viewport, it takes up all the available widht (and breaks correctly)

File: web/src/components/pages/dashboard/stats/leaderboard.css (hunk 1 above)
Comment (mritunjayz) [reply web/src/components/pages/dashboard/stats/leaderboard.css]: :+1:

File: web/src/components/pages/profile/avatar-setup/avatar-setup.css (hunk 2 above)
Comment (mritunjayz) [reply web/src/components/pages/profile/avatar-setup/avatar-setup.css]: Ok . I am looking.

**Comment by Gregoor** [comment]
Great, thanks for the fixes!

**Pull request merged**